### DISCUSSION TITLE
1.0.4 ED related?

### DISCUSSION THREAD

**Discussion by Res777** [rg35xx-cfw/Koriki#116] [General]
Is version 1.0.4 ED, which is online, related to the releases on this page, or not?

**Comment by souporman** [comment]
I know the archive.org upload says 1.0.4, but the actual files say 1.0.3 when you download it.  Seems like it's just somebody's fork of this work?

**Reply by italianbeef12** [reply]
Yeah, I think the spanish 1.04 is not official from here, otherwise it would be posted here.
